.flake8

```diff
@@ -33,12 +33,33 @@ exclude =
   # Metadata of `pip wheel` cmd is autogenerated
   pip-wheel-metadata,
 
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  I  # flake8-isort is drunk + we have isort integrated into pre-commit
+  WPS300  # local folder imports are needed
```

Comment:
Are you sure that comments on the same line work well?

Reply:
Yes, corresponding errors are ignored or what do you mean?

Reply:
Yes. I'm only asking because the configparser lib in Python has some funny bugs that make it behave dumb sometimes.